addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -649,6 +653,146 @@ QUnit.test('basic rendering of canceled notification', async function (assert) {
     );
 });
 
+QUnit.test('delete attachment should delete all text content free messages linked to it', async function (assert) {
```

Comment:
_"delete all attachments of message without content should no longer display the message"_